docs/operation/training.rst

```diff
@@ -0,0 +1,235 @@
+=======================
+Training and Evaluation
+=======================
+
+
+
+Virtual Environment Setup
+=========================
+
+Effective containerization is important when it comes to running machine learning models as there can be conflicting dependencies. You can either use a Virtual Environment or Conda.
```

Comment:
For paragraphs, it is normal to do one sentence per line. That way, a single small change at the beginning of a paragraph does not pollute the git history of the entire paragraph, just the sentence.

```suggestion
Effective containerization is important when it comes to running machine learning models as there can be conflicting dependencies.
You can either use a Virtual Environment or Conda.
```